src/makeRobotAccountant.js

```diff
@@ -7,6 +7,21 @@
 
 function makeRobotAccountant() {
   // write code here
+  let counter = 0;
+
+  return function getSum(a) {
+    return function(b) {
+      counter++;
+
+      if (counter <= 3) {
+        return a + b;
+      } else if (counter % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return a + b;
```

Comment:
IMO this logic could well be simplified, it is unnecesary to use else or else if.